### ISSUE TITLE
mythdb.txt file format

### ISSUE THREAD

**Issue by dmfrey** [thecount2a/mythtv-stream-mpeg-dash#2]
Can you add the format for this file?

**Comment by dmfrey** [comment]
there is a config.xml file in ~/.mythtv. Can that be used instead of a separate file?

**Comment by garretmcgraw** [comment]
Yes, that should be able to be used, if the web server user were allowed to read it.  On my system, that file is owned by mythtv and it is only readable by mythtv.  I would be welcome to another approach of getting the password such as the method mythweb uses (though apache config file) but that would require modifying the web server configuration which can be a pain.  The main reason I did that is so that I wouldn't be committing my database password to git but feel free to use any approach you want.

To answer your original question, the format of this file is just the mythtv database password in plain text (the database username and database name are variables in the index.php file).  I would welcome suggestions of a better approach.  My main goal would be to do a closer integration of this script as a mythweb plugin of some sort.  Then I wouldn't need to worry about passwords and such.  Unfortunately, I haven't found any way to implement mythweb plugins at this point.

**Comment by thecount2a** [comment]
Yes, that should be able to be used, if the web server user were allowed to read it. On my system, that file is owned by mythtv and it is only readable by mythtv. I would be welcome to another approach of getting the password such as the method mythweb uses (though apache config file) but that would require modifying the web server configuration which can be a pain. The main reason I did that is so that I wouldn't be committing my database password to git but feel free to use any approach you want.

To answer your original question, the format of this file is just the mythtv database password in plain text (the database username and database name are variables in the index.php file). I would welcome suggestions of a better approach. My main goal would be to do a closer integration of this script as a mythweb plugin of some sort. Then I wouldn't need to worry about passwords and such. Unfortunately, I haven't found any way to implement mythweb plugins at this point.
